### PR TITLE
change header value

### PR DESCRIPTION
**Pull request by SHREYANSH1814**

### CODE REVIEW

File: finalised/index.html [hunk 1]
```diff
@@ -1,154 +1,133 @@
...
     <div class="navbar">
-
-        <ul class="nav">
-            <li><a href="./index.html">Home</a></li>
-            <li><a href="#">About Us</a></li>
-            <li><a href="#">Contact</a></li>
-            <li><a href="#">More</a></li>
-            <li><a href="#">Features</a></li>
-        </ul>
+      <ul class="nav">
```
Comment (SHREYANSH1814) [inline finalised/index.html:17]: why did you add this /

**Review by abhi2k1 — CHANGES_REQUESTED** [review]
requested changes

**Review by abhi2k1 — APPROVED** [review]
